README.md

```diff
@@ -1,8 +1,152 @@
+[2022-01-13 14:08:16,261] INFO produced: StormDetailsTable              with formula sha ce78a5c3 and root versions 2011
+
+(venv) cabin $ bin/cabin status
+| version | table                             |   rows |    size | inputs               | outputs |
+|--------:|:----------------------------------|-------:|--------:|:---------------------|:--------|
+|    1  ✓ | StormDetailsTable::2011::ce78a5c3 | 79,101 | 19.4 MB | (1) StormDetailsFile | (0)     |
+```
+
+**If the root version were to change to 2012, we can update the dataset.** Note that:
+
+* a manual change to `cabin/datasets/StormDetails.py` is required to update the version of the source data from "2011" to "2012"
+* this change immediately renders the imported table (with data from 2011) outdated, reflected in status
+* the `import --all` triggers a new file to be downloaded from the external source to the local downloads dir
+* the the new table has different data content (see rows, and size in `status`)
+
+```
+(venv) cabin $ bin/cabin status
+| version | table                             |   rows |    size | inputs               | outputs |
+|--------:|:----------------------------------|-------:|--------:|:---------------------|:--------|
+|    1  ! | StormDetailsTable::2011::ce78a5c3 | 79,101 | 19.4 MB | (1) StormDetailsFile | (0)     |
+
+(venv) cabin $ bin/cabin import --all --dry-run
+[2022-01-13 14:11:38,443] INFO produce:  StormDetailsFile               with formula sha f9c219d7 and root versions 2012
+[2022-01-13 14:11:38,443] INFO produce:  StormDetailsTable              with formula sha 00ea4b32 and root versions 2012
+
+(venv) cabin$ bin/cabin import --all
+[2022-01-13 14:11:44,701] INFO produce:  StormDetailsFile               with formula sha f9c219d7 and root versions 2012
+downloads 100%[===================================================================>]  63.87M  6.52MB/s    in 11s     
+[2022-01-13 14:11:56,427] INFO Successfully downloaded to /Users/.../cabin/downloads/StormDetailsFile::2012::f9c219d7.csv
+[2022-01-13 14:11:56,429] INFO produced: StormDetailsFile               with formula sha f9c219d7 and root versions 2012
+[2022-01-13 14:11:56,429] INFO produce:  StormDetailsTable              with formula sha 00ea4b32 and root versions 2012
+[2022-01-13 14:12:06,402] INFO Imported 64508 rows to table: StormDetailsTable::2012::00ea4b32 
+[2022-01-13 14:12:06,447] INFO produced: StormDetailsTable              with formula sha 00ea4b32 and root versions 2012
+
+(venv) cabin$ bin/cabin status
+| version | table                             |   rows |    size | inputs               | outputs |
+|--------:|:----------------------------------|-------:|--------:|:---------------------|:--------|
+|    1  ! | StormDetailsTable::2011::ce78a5c3 | 79,101 | 19.4 MB | (1) StormDetailsFile | (0)     |
+|    2  ✓ | StormDetailsTable::2012::00ea4b32 | 64,508 | 15.2 MB | (1) StormDetailsFile | (0)     |
```

Comment:
Re verbosity: I'd skip re-import part of this demo. Once `cabin status` shows a dataset is out of date, the point has been made I think, ie I suggest keep lines 73-76, drop 77-96.